sdk/include/crt/string.h

```diff
@@ -405,6 +455,16 @@ extern "C" {
   strstr(
     _In_z_ const char *_Str,
     _In_z_ const char *_SubStr);
+_CRT_RESTORE_GCC_WARNINGS
+
+#ifdef __cplusplus
+    extern "C++"
+    _Check_return_ _Ret_maybenull_
+    inline char* __CRTDECL strstr(_In_z_ char* const _String, _In_z_ char const* const _SubString)
```

Comment:
I know it's the same, etc., but for consistency with the rest of the file, could you use `const char` instead of `char const` (and same for wchar_t in the other new functions)? (This is also the order in which these are mentioned in MSDN).

Reply:
In fact `char* const` is NOT the same as `const char*`! But the trailing const is pointless and rather confusing, so I removed it. And I changed `char const*` to `const char*` (which IS the same) for consistency.